tests/pulseaudio_dbus_spec.lua

```diff
@@ -189,4 +190,17 @@ b.describe("PulseAudio with DBus", function ()
                 	end
 		   		end
            end)
+
+		   b.it("Will set the next sink as the FallbackSink", function()
+		   	   if total_number_of_sinks <= 1 then
+				   print("\nWARNING: won't set the next sink as the FallbackSink because there is only one sink available in this machine")
+				   return
+			   end
+			   for s=1,total_number_of_sinks do
+			       if core.FallbackSink ~= sink[s] then
+					   core:set_fallback_sink(sink[s])
```

Comment:
We should assert that the sink has actually been set, also if we exit the loop without having set the new fallback sink we should fail the test.